webapp/package.json

```diff
@@ -35,18 +35,18 @@
     "babel-preset-react": "^6.24.1",
     "babel-preset-stage-0": "^6.24.1",
     "babel-register": "^6.24.1",
-    "css-loader": "^0.28.11",
+    "css-loader": "^3.0.0",
```

Comment:
I'm not too sure about that one. 

I think last time I hold on the update because in the release note it says: `minimum require webpack version is 4` (https://github.com/webpack-contrib/css-loader/compare/v0.28.11...v1.0.0).

I think it was working ok though...

Reply:
I quickly built and ran Mojito webapp and didn't see anything broken. This change was required to resolve 2 vulnerabilities:
```
# Run  npm install --save-dev css-loader@3.0.0  to resolve 2 vulnerabilities
SEMVER WARNING: Recommended action is a potentially breaking change
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Moderate      │ Denial of Service                                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ js-yaml                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ css-loader [dev]                                             │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ css-loader > cssnano > postcss-svgo > svgo > js-yaml         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/788                       │
└───────────────┴──────────────────────────────────────────────────────────────┘


┌───────────────┬──────────────────────────────────────────────────────────────┐
│ High          │ Code Injection                                               │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ js-yaml                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ css-loader [dev]                                             │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ css-loader > cssnano > postcss-svgo > svgo > js-yaml         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/813                       │
└───────────────┴──────────────────────────────────────────────────────────────┘
```

You can also try running `npm audit` from master branch. It gives you security report in the output with recommended changes.

Reply:
I guess if the build/app is working fine, it should be ok to upgrade. I was hoping the old version were going to be updated at some point to fix the security issues but if it is not the case I'm fine with this.